app/validations/site-validation.js

```diff
@@ -82,10 +82,15 @@ export const errorValidation = Joi.object()
       }),
     emailPrimary: Joi.string()
       .pattern(emailRegex)
-      .messages({ '*': 'Geef een geldig e-mailadres in' }),
-    websitePrimary: Joi.string().optional().pattern(websiteRegex).messages({
-      '*': 'Geef een geldig internetadres in',
-    }),
+      .optional()
+      .messages({ '*': 'Geef een geldig e-mailadres in' })
+      .allow(''),
+    websitePrimary: Joi.string()
+      .optional()
+      .pattern(websiteRegex)
+      .messages({
+        '*': 'Geef een geldig internetadres in'.allow(''),
```

Comment:
You have a typo here that gives me this error:
While generating link to route "sites.new": "Geef een geldig internetadres in".allow is not a function,
the allow should be in the next line